packages/flutter_tools/lib/src/commands/daemon.dart

```diff
@@ -1276,3 +1300,57 @@ class LaunchMode {
   @override
   String toString() => _value;
 }
+
+enum OperationType {
+  reloadMethod,
```

Comment:
you can remove reloadMethod - I'm going to be building this functionality into the regular hot reload instead

Reply:
That method is currently published [in the docs](https://github.com/flutter/flutter/blob/10571920c0fba5a56c4142b38df82d1671ae9f0b/packages/flutter_tools/doc/daemon.md#appreloadmethod) so removing it may be a breaking change? (I don't know who would've used it though)

Reply:
I would be surprised if anyone is using it, but it could be mapped to hot reload for backwards compat. Doesn't need to be solved in this PR.

Reply:
sgtm - I'll do in a separate PR once this one's done!

Reply:
I was taking a look at removing `reloadMethod`, but it looks like there is a lot of code relating to this (for ex. arguments passed to `device.connect` and VM service calls in `vmservice.dart`. It's not clear how much of this should go (whether it's just the daemon call, or everything related to it everywhere).

If it's easier for you to do this when you're building that functionality into regular hot reload, I'm happy to leave it for you 😄